huaweicloud/services/vpn/resource_huaweicloud_vpn_connection.go

```diff
@@ -386,25 +386,25 @@ func resourceConnectionCreate(ctx context.Context, d *schema.ResourceData, meta
 
 	id, err := jmespath.Search("vpn_connection.id", createConnectionRespBody)
 	if err != nil {
-		return diag.Errorf("error creating Connection: ID is not found in API response")
+		return diag.Errorf("error creating VPN connection: ID is not found in API response")
 	}
 	d.SetId(id.(string))
 
 	err = createConnectionWaitingForStateCompleted(ctx, d, meta, d.Timeout(schema.TimeoutCreate))
 	if err != nil {
-		return diag.Errorf("error waiting for the Create of Connection (%s) to complete: %s", d.Id(), err)
+		return diag.Errorf("error waiting for the creating VPN connection (%s) to complete: %s", d.Id(), err)
```

Comment:
```suggestion
		return diag.Errorf("error waiting for creating VPN connection (%s) to complete: %s", d.Id(), err)
```
and fix other places